docs/book/src/getting-started/testdata/project/Makefile

```diff
@@ -78,12 +78,16 @@ test-e2e: manifests generate fmt vet ## Run the e2e tests. Expected an isolated
 	}
 	go test ./test/e2e/ -v -ginkgo.v
 
+.PHONY: lint-config
+lint-config: golangci-lint ## Verify golangci-lint linter configuration
+	$(GOLANGCI_LINT) config verify
+
 .PHONY: lint
-lint: golangci-lint ## Run golangci-lint linter
+lint: lint-config ## Run golangci-lint linter
 	$(GOLANGCI_LINT) run
 
 .PHONY: lint-fix
-lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
+lint-fix: lint-config ## Run golangci-lint linter and perform fixes
```

Comment:
Same here
And we need to have golangci-lint to ensure that the bin will be downloaded